web-previewer/src/GenesisContract.json

```diff
@@ -0,0 +1,632 @@
+{
```

Comment:
Suggestion: Perhaps we could move this to `src/contracts/GenesisContract.json`

Reply:
good idea